utils/errorHandler.ts

```diff
@@ -0,0 +1,54 @@
+import {AxiosError} from 'axios';
+import {APIError} from './interfaces';
+import Toast from 'react-native-toast-message';
+import {StyleSheet} from 'react-native';
+
+export default function errorHandler(error: AxiosError<APIError>) {
+  const heading = 'An Error Occured!';
+  if (error.response) {
+    const data = error.response.data;
+
+    console.log(error.response.data);
+    console.log(error.response.status);
+    console.log(error.response.headers);
+
+    if (data) {
+      Toast.show({
+        type: 'error',
+        text1: heading,
+        text2: data.message,
+        text1Style: styles.text1Style,
+        text2Style: styles.text2Style,
+      });
+    }
+  } else if (error.request) {
+    console.log(error.request);
+
+    Toast.show({
```

Comment:
instead of passing styles you should be create variants for a toast for example, variant = 'info' will should different text style and variant= 'success' should show a different text style